src/toxcore/dht/server/mod.rs

```diff
@@ -626,102 +626,39 @@ impl Server {
     /// Function to handle incoming packets and send responses if necessary.
     pub fn handle_packet(&self, packet: Packet, addr: SocketAddr) -> impl Future<Item = (), Error = Error> + Send {
         match packet {
-            Packet::PingRequest(packet) => {
-                debug!("Received ping request");
```

Comment:
Why is this useless?

Reply:
As I wrote, it duplicates the log message here: https://github.com/tox-rs/tox/blob/master/src/toxcore/dht/server_ext.rs#L43

Reply:
Ah Thanks!